mapsforge-map/src/main/java/org/mapsforge/map/model/MapViewPosition.java

```diff
@@ -250,16 +264,6 @@ public synchronized void init(PreferencesFacade preferencesFacade) {
     }
 
     /**
-     * Animates the center position of the map by the given amount of pixels.
-     *
-     * @param moveHorizontal the amount of pixels to move this MapViewPosition horizontally.
-     * @param moveVertical   the amount of pixels to move this MapViewPosition vertically.
-     */
-    public void moveCenter(double moveHorizontal, double moveVertical) {
-        this.moveCenterAndZoom(moveHorizontal, moveVertical, (byte) 0, true);
-    }
-
-    /**
```

Comment:
Why different implementation specifically for `moveCenter(moveHorizontal, moveVertical)` ?